app/lib/common/widget/ChatOverview.dart

```diff
@@ -1,12 +1,11 @@
 // ignore_for_file: prefer_const_constructors, avoid_unnecessary_containers
 
-
 import 'package:effektio/common/widget/AppCommon.dart';
 import 'package:effektio/common/widget/customAvatar.dart';
 import 'package:effektio/screens/HomeScreens/ChatScreen.dart';
 import 'package:effektio_flutter_sdk/effektio_flutter_sdk_ffi.dart';
 import 'package:flutter/material.dart';
-
+import 'package:intl/intl.dart';
```

Comment:
don't we also have to add `intl` to the pubspec? Or is that always included?

Reply:
It is already there in pubspec, was added during localization work.